sysdig/internal/client/secure/models.go

```diff
@@ -125,7 +125,7 @@ type notificationChannelWrapper struct {
 type Rule struct {
 	ID          int      `json:"id,omitempty"`
 	Name        string   `json:"name"`
```

Comment:
Should name also add the omitempty tag as it is also not allowed to be empty?

Reply:
I think this might not be needed as backend needs to throw an error on empty rule name, so sending "" should be fine.